_pages/en/technical-platform.md

```diff
@@ -86,7 +86,15 @@ More detailed Azure DevOps Pipeline doc: [Azure Pipeline](../assets/docs/azure-p
 
 ## Security Implications
 
-- Things we have done to make it more secure.
+- Using GraphQL mobible friendly API with strong input validation to prevent hacking the database.
+- Using containers to minimize attack surface--less code bug, less chance to be attacked. 
+- Continuous deployement let us fix security problems quickly- pipleline has "npm install",  will check all the dependence security issue. 
+- Scrambling personal information in both protypes before saving to database.
+- Only the delivery team members can access to the data which gathered from protypes testing. 
```

Comment:
Suggest reviewing this to be more clear. Right now, CDS delivery team members can't access data only RCMP team members. Need a way to phrase it that only authorized people can access data?

Reply:
good suggestion